opennebula/resource_opennebula_virtual_machine.go

```diff
@@ -266,6 +286,29 @@ func resourceOpennebulaVirtualMachineCreate(d *schema.ResourceData, meta interfa
 		}
 	}
 
+	// If the user requested a desired state except running at creation, handle this request after starting the VM
+	desiredState := d.Get("desired_state").(string)
+	switch desiredState {
+	case "running":
```

Comment:
it seems it is not respecting go fmt rules

Reply:
What do you mean exactly? My IDE doesn't show an error. Is the empty case the problem?